src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.data.person;
+
```

Comment:
Oh, I believe you are referring to header comments. Thanks, will remember it next time.